test/environments/norails/Gemfile

```diff
@@ -22,3 +22,8 @@ gem 'simplecov' if ENV['VERBOSE_TEST_OUTPUT']
 
 gem 'warning'
 gem 'loofah', '~> 2.20.0' if RUBY_VERSION >= '2.4.0' && RUBY_VERSION < '2.5.0'
+
+if RUBY_VERSION.split('.')[0..1].join('.').to_f >= 3.4
```

Comment:
Are these environments able to use the `ruby_verison_float` method?

Also, it seems a little sacrilegious, but what do you think about using the `Gem::Version.new` API to compare Ruby versions? Ruby language versions seem to follow the same process as gem versions. However, it won't allow you to use a float, you'd have to use a string.

`Gem::Version.new(RUBY_VERSION) >= '3.4'`

Reply:
> Are these environments able to use the ruby_verison_float method?

Given that these are direct entrypoints (as opposed to the `Envfile` files), I didn't want to drag in any dependencies.

> you'd have to use a string

Yeah, string comparison for math isn't ideal even though we've lucked out with Ruby v1 and v2 never having gone to double digits with the minor version.

Maybe in future we could author a helper method that simply calls `Gem::Version.new` on both `RUBY_VERSION` and the evaluated Ruby version string. I think we've done that in some places previously inline (without using a method).